resources/assets/js/components/JobBuilder/JobBuilderStep.tsx

```diff
@@ -133,14 +133,17 @@ const JobBuilderStep: React.FunctionComponent<JobBuilderStepProps> = ({
 
   return (
     <section>
-      <JobBuilderProgressTracker
-        job={job}
-        tasks={keyTasks}
-        maxTasksCount={VALID_COUNT}
-        criteria={criteria}
-        dataIsLoading={dataIsLoading}
-        currentPage={currentPage}
-      />
+      {jobId !== null && (
```

Comment:
This stops the progress bar from appearing on the first step, when you're creating the poster.

Reply:
fixed